CommonAssemblyInfo.cs

```diff
@@ -1,4 +1,4 @@
 using System.Reflection;
 
-[assembly: AssemblyVersion("6.5.0")]
-[assembly: AssemblyFileVersion("6.5.0")]
+[assembly: AssemblyVersion("6.90.0")]
```

Comment:
Hmm, I thought I did that, but maybe I did something wrong. How exactly should I do so?